distribution/lib/Standard/Database/0.0.0-dev/src/Connection/PostgreSQL.enso

```diff
@@ -0,0 +1,81 @@
+from Standard.Base import all
+
+import Standard.Database.Data.Dialect
+import Standard.Database.Connection.Connection
+from Standard.Database.Connection.Credentials as Credentials_Module import Credentials
+import Standard.Database.Connection.Connection_Options
+import Standard.Database.Connection.SSL_Mode
+from Standard.Database.Connection.SSL_Mode import all
+import Standard.Database.Connection.Client_Certificate
+
+polyglot java import org.postgresql.Driver
+
+type PostgreSQL
+    ## Connect to a PostgreSQL database.
+
+       Arguments:
+       - host: The hostname of the database server (defaults to localhost).
+       - port: The port of the database server (defaults to 5432).
+       - database: The database to connect to (if not provided or empty, the default database will be used).
+       - credentials: The credentials to use for the connection (defaults to PGPass or No Authentication).
+       - use_ssl: Whether to use SSL (defaults to `Prefer`).
+       - client_cert: The client certificate to use or `Nothing` if not needed.
+    type PostgreSQL (host:Text='localhost') (port:Integer=5432) (database:Text='postgres') (credentials:(Credentials|Nothing)=Nothing) (use_ssl:SSLMode=Prefer) (client_cert:(Client_Certificate|Nothing)=Nothing)
+
+    ## Build the Connection resource.
+
+       Arguments:
+       - options: Overrides for the connection properties.
+    connect : Connection_Options
+    connect options =
+        if Driver.isRegistered.not then Driver.register
+
+        properties = options.merge self.jdbc_properties
+        Connection.create_jdbc_connection self.jdbc_url properties self.dialect
+
+    ## Provides the jdbc url for the connection.
+    jdbc_url : Text
+    jdbc_url =
+        'jdbc:postgresql://' + self.host + ':' + self.port.to_text + (if self.database == '' then '' else '/' + self.database)
+
+    ## Provides the properties for the connection.
+    jdbc_properties : [Pair Text Text]
+    jdbc_properties =
+        credentials = case self.credentials of
+            Nothing -> PostgreSQL.read_pgpass self.host self.port self.database
+            Credentials username password ->
+                [ Pair 'user' username, Pair 'password' password ]
```

Comment:
I've never seen this kind of usage of whitespace in our code before.
```suggestion
                [Pair 'user' username, Pair 'password' password]
```

I'm not saying it's wrong, but I've never seen it and below we also don't have the additional whitespace, I think we should be consistent.